connector/connector.js

```diff
@@ -3,6 +3,7 @@ const app = require('../server/server.js');
 const connector = app.dataSources.db.connector;
 const DataSourceHandler = require('./data-source-handler');
 const clone = require('lodash').clone;
+const FacetHandler = require('./facet-handler');
 const MiddlewareHandler = require('./middleware-handler');
 const ModelHandler = require('./model-handler');
 const RelationsHandler = require('./relation-handler');
```

Comment:
Possibly we could follow lexical order; please see https://github.com/strongloop/loopback/issues/2949 ; however please feel free to ignore this since the convention is not official yet.